test/choose_watt_tests/choose_watt_tests.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+from ruamel.yaml import YAML
+import subprocess
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--changed-workflows", help="Name of workflow(s) which have changed", nargs='*')
+parser.add_argument("--changed-jsons", help="Name of json(s) which have changed", nargs='*')
+parser.add_argument("--watt-config", help="WATT config yaml")
+parser.add_argument("--womtool-jar")
+
+#copied from watt
```

Comment:
yeah, I think that could be a better option, especially because then things like running only the specific changed test when a json changes would become much simpler.

It would add some additional complexity to watt though.  I'm envisioning arguments `--changed-workflows` and `changed-jsons` in watt, which would be mutually exclusive with the `--workflow` and `--test` arguments.  I suppose they wouldn't technically need to be mutually exclusive, but I think if they weren't we could end up with some fairly unintuitive behavior, without much valuable gain in functionality, imo.  It would also add a requirement of a womtool jar, in addition to a cromwell jar.  

My preference would be to get this merged this way first, and then we can explore porting this functionality to watt if we decide it makes sense.

Reply:
If it were to merge into watt, I'd imagine it would make sense to have two subcommands: `test` and `dep` or something like that. For now we can merge it here and think about it later.